### DISCUSSION TITLE
Vector3 constructor for singular float

### DISCUSSION THREAD

**Discussion by FireCatMagic** [godotengine/godot-proposals#7312] [Scripting]
Current Vector3 Constructors:
![image](https://github.com/godotengine/godot-proposals/assets/40964335/23a14919-30d5-4b5b-afb5-c8ddda1855c3)

Proposed Constructor (Inspect Element)
![image](https://github.com/godotengine/godot-proposals/assets/40964335/a7d7d0e3-ff0b-4415-9a93-d26cdc137903)

Using this constructor would be equivalent to using the float for x, y and z

Example
Vector3(4.5)     =        Vector3(4.5, 4.5, 4.5)

Why?
Just a miniscule change that wouldn't be difficult to implement. It would be useful for scenarios like the one that I have here:
![image](https://github.com/godotengine/godot-proposals/assets/40964335/97c7afcc-550c-4f73-a92c-3d113509b849)

**Comment by theraot** [comment]
While I'm sympathetic to the idea, I want to point out that you can do this:

```
Vector3.ONE * 4.5
```

Or this:

```
Vector3.ONE * properties.power * normal
```

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/4822.
